graphdatascience/tests/integration/conftest.py

```diff
@@ -55,7 +55,11 @@ def gds_with_tls(request: FixtureRequest) -> GraphDataScience:
         root_ca = f.read()
 
     _gds = GraphDataScience(
-        URI_TLS, auth=AUTH, arrow=True, arrow_disable_server_verification=True, arrow_tls_root_certs=root_ca
+        URI_TLS,
+        auth=AUTH,
+        arrow=True,
+        arrow_disable_server_verification=True,
+        arrow_tls_root_certs=root_ca,
```

Comment:
Weird that it didn't format that before. Is it because it's in test scope?

Reply:
The previous formatting was fine according to the rules of `black` I think. I just changed it for personal preference. It still obeys the rule we've set up